controllers/vaultpkisecret_controller.go

```diff
@@ -183,9 +184,14 @@ func (r *VaultPKISecretReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		if err := r.updateStatus(ctx, o); err != nil {
 			return ctrl.Result{}, err
 		}
+
+		r.SyncRegistry.Add(req.NamespacedName)
```

Comment:
It looks like the sync reason may need to be updated on line 123? I see the VDS controller uses "force sync" for this case.
https://github.com/hashicorp/vault-secrets-operator/blob/697604208121ce6e38fbad214538700ddec1b855/controllers/vaultpkisecret_controller.go#L122-L123

Reply:
Synced up in d10ffde439745a2ad2b55d1170e7cce99c10440a